python-core-developer.md

```diff
@@ -0,0 +1,90 @@
+# Python Core Developer
+
+Within the first 6 months, you will design, prototype, implement and deploy one
+or more major mission-critical services underlying Ludia's game application.
+The process will include developing an early understanding of our core logic
+infrastructure, documenting design and code details and participating in daily
+peer code-reviews.
+
+* Write maintainable code with extensive test coverage, working in a
+  professional software engineering environment (with source control,
+  dev/test/stage/prod release cycle, continuous deployment) - cowboy coders
+  need not apply;
+* Collaborate across the company and interact with our customers (game
+  developers, data scientists...) to define, design and built new APIs;
+* Tackle complex problems in distributed computing, virtualization performance,
+  distributed queuing, real-time multiplayer logic, and more;
+* Maintain and evelove legacy code to keep compliancy with third parties.
+
+## Requirements
+
+We places a high premium on engineers with a deep commitment to quality and a
+demonstrated understanding of underlying technologies. To apply for the role,
+you should have some or all of the following:
+
+* A Bachelors or preferably Masters degree in Computer Science or Computer
+  Engineering, or equivalent experience;
+* 4+ Years of engineering experience;
+* Solid knowledge of python;
+* Solid knowledge of computer science theory
+  (design pattern/algorithms/software engineering);
+* Solid knowledge of web technologies;
+* Solid knowledge of unit and integration testing methodologies, and the
+  ability to write, debug and deploy testing frameworks;
+* Experience building and scaling distributed, highly available systems;
+* Experience with relational and non-relational databases;
+* Experience with UNIX-like/Linux environment;
+* Experience with modern Data Analysis ecosystem.
+
+### Bonus Points
+
+* Knowledge of virtualenv/pyramid/sqlAlchemy/alembic/angularjs/spark/AWS;
+* Experience with mobile development;
+* Experience with APIs;
+* Knowledge of modern javascript tools;
+* Notable contributions to open source software projects;
+* Good communication, organization and project management skills;
+* Curious and autonomous.
+
+# Core Développeur Python
+
+Dans les 6 premiers mois, vous devez concevoir, prototyper, mettre en œuvre et
+déployer un ou plusieurs services cruciaux pour les jeux de Ludia. Le processus
+comprend:
+
+* L'apprentissage de notre infrastructure;
+* La documentation des APIs;
+* Les revues quotidiennes de code;
+* L'écriture de code maintenable avec une large couverture de test, dans un
+  environnement d'ingénierie de logiciels professionnels - Les cow-boys ne
+  doivent pas appliquer;
```

Comment:
Bad translation from english: cowboy coders should not apply. 

J'enlève anyway.